third_party/pyth/price-service/README.md

```diff
@@ -1,54 +1,58 @@
 # Pyth Price Service
 
-This service exposes a REST and WS api to provide latest attestation message for a price feed id.
+Pyth price service is a webservice that listens to the Wormhole Network for Pyth price updates and serves them via a
+convenient web API. The service allows users to easily query for recent price updates via a REST API, or subscribe to
+a websocket for streaming updates. [The Pyth Network Javascript SDKs](https://github.com/pyth-network/pyth-js) connect
+to an instance of the price service in order to fetch on-demand price updates.
 
-## Build
+## Public Endpoints
 
-First, build the wasm container from the repo root:
+The Pyth Data Association operates two public endpoints for the price service, for mainnet and testnet respectively.
+These endpoints can be used to test integrations with Pyth Network:
 
-```
-docker buildx build -f Dockerfile.wasm -o type=local,dest=. .
-```
+| network | url                             |
+| ------- | ------------------------------- |
+| mainnet | https://xc-mainnet.pyth.network |
+| testnet | https://xc-testnet.pyth.network |
 
-Then, build the pyth_price_service docker container from the repo root:
+For production deployments, developers integrating with Pyth Network are **strongly encouraged** to host their own instance of the price service for maximum resilience and decentralization.
+By running an independent instance of this service, developers tap directly into Wormhole's peer-to-peer network to stream Pyth price updates.
+This peer-to-peer network has built-in redundancy and is therefore inherently more reliable than a centralized service operated by the PDA.
 
-```
-$ docker buildx build -f third_party/pyth/price-service/Dockerfile.price_service -t pyth_price_service .
-```
+## Wormhole Spy
 
-Run the spy_guardian docker container in TestNet:
+Price service depends on a Wormhole spy to stream Pyth messages from the Wormhole Network to it.
+[Spy](https://github.com/wormhole-foundation/wormhole/blob/main/node/cmd/spy/spy.go) is a component of a
+Wormhole guardian node that joins the Wormhole Network peer-to-peer network and listens to the Wormhole verified
+messages and streams the messages that are coming from certain emitters (e.g., Pyth data emitters) to its subscribers.
```

Comment:
I think you could convey the same information here in fewer words. Try to go through and simplify the sentences to their essence.

Reply:
Also "The price service" and "The Spy" at the beginning of these sentences.

Reply:
maybe add a second paragraph here explaining how the price service connects to the spy and how it's configured